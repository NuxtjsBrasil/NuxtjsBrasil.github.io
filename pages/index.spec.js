import { mount } from '@vue/test-utils'
import Home from './index.vue'

describe('Home', () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(Home)
  })

  it('should match with snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
