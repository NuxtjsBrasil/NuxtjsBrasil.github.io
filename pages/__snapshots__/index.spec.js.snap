// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match with snapshot 1`] = `
<div>
  <section class="Hero">
    <div class="Hero__inner">
      <div class="Hero__logo">
        <!---->
        <h1>NUXT<span>BRASIL</span></h1>
      </div>
      <p class="Hero__lead">Grupo BR do NuxtJS
        <br>Universal Vue.js Applications</p>
      <div class="Hero__buttons">
        <a href="https://github.com/nuxtjsbrasil" target="_blank" class="Button Button--primary">Github Nuxt Brasil</a>
        <a href="https://nuxtjs.org/" target="_blank" class="Button Button--secondary">Nuxtjs.org (en)</a>
      </div>
    </div>
  </section>
</div>
`;
